samples/nrf9160/gnss/src/main.c

```diff
@@ -278,21 +269,67 @@ static int gnss_init_and_start(void)
 		return -1;
 	}
 
-	if (nrf_modem_gnss_nmea_mask_set(NRF_MODEM_GNSS_NMEA_RMC_MASK |
-					 NRF_MODEM_GNSS_NMEA_GGA_MASK |
-					 NRF_MODEM_GNSS_NMEA_GLL_MASK |
-					 NRF_MODEM_GNSS_NMEA_GSA_MASK |
-					 NRF_MODEM_GNSS_NMEA_GSV_MASK) != 0) {
+	/* Enable all NMEAs. */
```

Comment:
Nitpicking: "Enable all supported NMEA messages."

Reply:
True, I'll update it.